### ISSUE TITLE
HMR + InProcess is not working

### ISSUE THREAD

**Issue by cjblomqvist** [dotnet/aspnetcore#13382]
### Describe the bug
See https://github.com/aspnet/AspNetCore/issues/7253

### To Reproduce
See https://github.com/aspnet/AspNetCore/issues/7253

I'm using: 
* Visual Studio 16.2.3
* .NET Core SDK 2.2.401
* aspnet-webpack 3.0.0
* webpack-hot-middleware 2.25.0

There's also a repro repo at https://github.com/paulirwin/WebpackHMRIssue - albeit it doesn't necessarily have the exact same version of all packages.

### Expected behavior
See https://github.com/aspnet/AspNetCore/issues/7253

### Screenshots
See https://github.com/aspnet/AspNetCore/issues/7253

### Additional context
See https://github.com/aspnet/AspNetCore/issues/7253

Basically, I'm just doing what https://github.com/aspnet/AspNetCore/issues/7253#issuecomment-521686732 said and creating a new issue since it seems the original issue will not be re-opened.

**Comment by Tratcher** [comment]
@jkotalik

**Comment by jkotalik** [comment]
@BillHiebert you recently fixed an issue with this before, correct?

**Comment by anurse** [comment]
There was some confusion here between hot-reload of the server-side code for IIS-hosted apps and the Hot Module Replacement feature of the SPA Services package, which is what caused us to believe this issue was "fixed" earlier.

The latter (HMR in SPA Services) is deprecated in 3.0 as per https://github.com/aspnet/Announcements/issues/379 and no longer supported. The discussion thread (https://github.com/dotnet/aspnetcore/issues/12890) has some further details on migration options.

Closing as this component is obsolete and no longer supported.
